src/ui/index.js

```diff
@@ -52,6 +59,9 @@ export default async (kbnServer, server, config) => {
       const app = uiExports.apps.byId[id];
       if (!app) return reply(Boom.notFound('Unknown app ' + id));
 
+      const acceptLanguageStr = req.headers['accept-language'];
```

Comment:
does this behave properly in the face of a client who doesn't pass this header in? Should any errs be caught here?